vision/tests/system.py

```diff
@@ -34,6 +35,7 @@
 LOGO_FILE = os.path.join(_SYS_TESTS_DIR, "data", "logo.png")
 PDF_FILE = os.path.join(_SYS_TESTS_DIR, "data", "pdf_test.pdf")
 PROJECT_ID = os.environ.get("PROJECT_ID")
+PROJECT_OUTSIDE = os.environ.get("GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT")
```

Comment:
Should we be setting this environment variable in our tests environment? Or are these tests being run somewhere else?

Reply:
The VPC SC test is being run elsewhere. No need for you to set this environment variable.